save-backend/src/main/kotlin/com/saveourtool/save/backend/storage/AvatarStorage.kt

```diff
@@ -47,21 +62,21 @@ class AvatarStorage(configProperties: ConfigProperties) :
 
 /**
  * @property type
+ * @property folderName
  * @property imageName
- * @property filename
  */
 data class AvatarKey(
     val type: AvatarType,
+    val folderName: String,
```

Comment:
can we rename it to 'objectName' for example. because folder name is implementation details which can be hidden here (it will be irrelevant after migration to S3)

Reply:
Ok. Fixed it